docs/api/browser-window.md

```diff
@@ -457,16 +457,19 @@ height areas you have within the overall content view.
 * `width` Integer
 * `height` Integer
 
+`animate` Boolean (optional) _OS X_
```

Comment:
Ah I didn't notice that, actually the `options` should also be in the list, it was a mistake that the `options` is not.

```markdown
* options
  * x
  * y
  * width
  * height
* animate
```